Resources/Locale/en-US/lathe/components/lathe-component.ftl

```diff
@@ -1,2 +1,3 @@
 lathe-component-upgrade-speed = speed
 lathe-component-upgrade-material-use = material use
+lathe-component-got-emmaged = Something zaps inside the machine
```

Comment:
like i said, you can remove this because EmagSystem handles this already when used properly.